docs/framework/wpf/getting-started/walkthrough-my-first-wpf-desktop-application.md

```diff
@@ -412,7 +420,7 @@ In this section, you'll create the XML data that is bound to various controls.
     [!code-xaml[ExpenseIt#23](~/samples/snippets/csharp/VS_Snippets_Wpf/ExpenseIt/CSharp/ExpenseIt8/ExpenseItHome.xaml#23)]
```

Comment:
These three code references should be combined. See <https://github.com/dotnet/docs/issues/10241>.